.github/workflows/release-test.yml

```diff
@@ -14,6 +14,6 @@ on:
 jobs:
   test:
     if: github.repository == 'remix-run/remix'
-    uses: remix-run/remix/.github/workflows/reusable-test.yml@main
+    uses: ./.github/workflows/reusable-test.yml
```

Comment:
keeping the other workflows as is for now as it explicitly states where a reusable workflow is running